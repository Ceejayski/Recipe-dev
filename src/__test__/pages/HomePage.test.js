import { Provider } from 'react-redux';
import { create } from 'react-test-renderer';
import HomePage from '../../pages/homePage';
import store from '../../redux';

test('renders Correctly', () => {
  const Placeholder = () => (
    <Provider store={store}>
      <HomePage />
    </Provider>
  );
  const tree = create(<Placeholder />).toJSON();
  expect(tree).toMatchSnapshot();
});
