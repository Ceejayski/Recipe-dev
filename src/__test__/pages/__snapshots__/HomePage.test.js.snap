// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders Correctly 1`] = `
Array [
  <div
    className="swiper-container mySwiper"
  >
    <div
      className="swiper-button-prev"
    />
    <div
      className="swiper-button-next"
    />
    <div
      className="swiper-pagination"
    />
    <div
      className="swiper-wrapper"
    >
      <div
        className="swiper-slide"
      >
        <img
          alt="slide1"
          src="slide1.jpg"
        />
        <div
          className="position-absolute slide-text"
        >
          <h1>
            RecipeDev
          </h1>
          <p>
            by a Dev turned Cook
          </p>
        </div>
      </div>
      <div
        className="swiper-slide"
      >
        <img
          alt="slide2"
          src="burger.jpg"
        />
        <div
          className="position-absolute slide-text"
        >
          <h1>
            Fast Food Recipes
          </h1>
          <p
            className="text-white"
          >
            like you have never had before
          </p>
        </div>
      </div>
      <div
        className="swiper-slide"
      >
        <img
          alt="slide3"
          src="dessert.jpg"
        />
        <div
          className="position-absolute slide-text"
        >
          <h1>
            Dessert Recipes
          </h1>
          <p
            className="text-white"
          >
            Sweettooth?... We have got you 
          </p>
        </div>
      </div>
      <div
        className="swiper-slide"
      >
        <img
          alt="slide4"
          src="pizza.jpg"
        />
        <div
          className="position-absolute slide-text"
        >
          <h1>
            Pizza Recipes
          </h1>
          <p
            className="text-white"
          >
            We Leave in a world were pizza gets home before the police do
          </p>
        </div>
      </div>
    </div>
  </div>,
  <div
    className="container"
  >
    <h3
      className="mt-5 text-center"
    >
      Recipes of the Day
    </h3>
    <div
      className="row"
    />
  </div>,
]
`;
